.github/workflows/check_local_dependencies.yml

```
@@ -0,0 +1,46 @@
+name: Build and Publish Artifact
+
+on:
+  workflow_call:
+    inputs:
+      python-version:
+        type: string
+        default: "3.7"
+    secrets:
+      JFROG_ARTIFACTORY_USER:
+        required: false
+      JFROG_ARTIFACTORY_PASSWORD:
+        required: false
+      JFROG_ARTIFACTORY_URL:
+        required: false
+
+jobs:
+  check-local-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+        with:
+          # checkout to a directory other than the current one
+          path: artifact_code
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ inputs.python-version }}
+          cache: "pip"
+
+      - name: Check local dependencies
+        env:
+          JFROG_ARTIFACTORY_URL: ${{ secrets.JFROG_ARTIFACTORY_URL }}
+          JFROG_ARTIFACTORY_USER: ${{ secrets.JFROG_ARTIFACTORY_USER }}
+          JFROG_ARTIFACTORY_PASSWORD: ${{ secrets.JFROG_ARTIFACTORY_PASSWORD }}
+        run: |
+          if [[ -n $JFROG_ARTIFACTORY_URL ]] && [[ -n $JFROG_ARTIFACTORY_USER ]] && [[ -n $JFROG_ARTIFACTORY_PASSWORD ]]; then
```

Let's remove this check and let this fail if the JFrog credentials are not set

---

Updated.